### PR TITLE
Removes pariah gun code and updates tg gun code

### PR DESCRIPTION
**Pull request by DTraitor**
## About The Pull Request
Pariah Station had it's own gun code. Highly modified tg to be exact. I removed it and updated the gun code to the latest tg. It fixed some bugs and also made the code a lot more maintainable.

## Why It's Good For The Game
`LessBadCode == Good`
Also fixes a bug that didn't add action button that switched fire modes.

## Changelog

:cl:
refactor: Replaced Pariah gun code with tg gun code.
/:cl:

### CODE REVIEW

File: code/datums/elements/weapon_description.dm [hunk 1]
```diff
@@ -0,0 +1,128 @@
+#define HITS_TO_CRIT(damage) round(100 / damage, 0.1)
+/**
+ *
+ * The purpose of this element is to widely provide the ability to examine an object and determine its stats, with the ability to add
+ * additional notes or information based on type or other factors
+ *
+ */
+/datum/element/weapon_description
+	element_flags = ELEMENT_BESPOKE
+	id_arg_index = 2
```
Comment (Kapu1178) [inline code/datums/elements/weapon_description.dm:10]: weapon descriptions were explicitly removed

**Review by Kapu1178 — CHANGES_REQUESTED** [review]

**Comment by Kapu1178** [comment]
will tm: 👍

File: code/modules/projectiles/guns/energy/laser.dm [hunk 2]
```diff
@@ -15,17 +15,18 @@
...
 	icon_state = "retro"
-	desc = "An older model of the basic lasergun, no longer used by Daedalus' private security. Nevertheless, it is still quite deadly and easy to maintain, making it a favorite amongst pirates and other outlaws."
+	desc = "An older model of the basic lasergun, no longer used by Nanotrasen's private security or military forces. Nevertheless, it is still quite deadly and easy to maintain, making it a favorite amongst pirates and other outlaws."
 	ammo_x_offset = 3
 
 /obj/item/gun/energy/laser/retro/old
 	name ="laser gun"
 	icon_state = "retro"
-	desc = "First generation lasergun, developed by MEO. Suffers from ammo issues but its unique ability to recharge its ammo without the need of a magazine helps compensate. You really hope someone has developed a better lasergun while you were in cryo."
+	desc = "First generation lasergun, developed by Nanotrasen. Suffers from ammo issues but its unique ability to recharge its ammo without the need of a magazine helps compensate. You really hope someone has developed a better lasergun while you were in cryo."
```
Comment (gonenoculer5) [inline code/modules/projectiles/guns/energy/laser.dm:29]: Ditto.

**Review by gonenoculer5 — CHANGES_REQUESTED** [review]
I might've missed a few things here but anything relating to a mention of NT in any fashion needs to be reverted back to what it was. A few other questions though would be why the detectives revolver was renamed to C38 if you know, probably just a TG-ism if I had to guess. I also dont personally really like the use of so many balloon alerts, as I believe to_chat() is more preferable when balloon alerts look so terrible. This obviously comes down to more of being Kapu's choice, but I thought I'd throw my hat in the ring.

**Review by gonenoculer5 — COMMENTED** [review]
Looks good now. Up to Kapu if he wants to move towards using balloon alerts, or keep using to_chat() as we did originally. Personally I prefer to_chat() over balloon's, but its his and frans call.

**Comment by Kapu1178** [comment]
YOLO MERGEEEEE

**Pull request merged**